demo/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,8 @@
+const pkg = require('../../package.json');
+const dateFormat = require('dateformat');
+
+export const environment = {
+  production: true,
+  version: JSON.stringify(pkg.version),
+  now: JSON.stringify(dateFormat(new Date(), 'dd mmm yyyy')),
```

Comment:
Since we use v5, can you try to do the format using date pipe, so we can drop `dateformat` dependency.